module1.js

```diff
@@ -69,13 +64,19 @@ function question1(name1) {
   @returns [array 0 ... 100] 
 */
 
-// function question4() {
-//   // TODO
-//   for ( /* ??? */ ) {
-//     // TODO
-//   }
-//   return /* ??? */;
-// }
+
+function question4(){
+  arr=[]
+  for(let i=0;i<101;i++){
+    arr.push(i);
+  }
+  newArr=[];
+  arr.forEach(function(item){
+    if(item%2===0)
+    newArr.push(item);
+  })
+  return newArr;
+}
 
```

Comment:
You can shorten this solution up a bit, if you increment i by 2 instead of 1, like so...
```
for (var i = 0; i < 100; i+=2)
```
that way, you wouldn't need to use the `if` statement, and you would also remove the need for the newArr variable.